src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3453,20 +3488,7 @@ public override BoundNode VisitAddressOfOperator(BoundAddressOfOperator node)
         /// </summary>
         private bool ReportNullReferenceArgumentIfNecessary(BoundExpression argument, TypeSymbolWithAnnotations argumentType, ParameterSymbol parameter, TypeSymbolWithAnnotations paramType)
```

Comment:
Should we consider removing this method altogether and having callers just call the underlying version?

Reply:
I'll leave that for a follow-up (Chuck is doing further simplifications).